File: Algorithms/Python/bubbleSort.py
# bubble sort

# test numbers
toSort = [3, 7, 2, 6, 4]

# perform bubble sort
for i in range(5):
    for j in range(4):
        if toSort[j] > toSort[j+1]:
            temp = toSort[j]
            toSort[j] = toSort[j + 1]
            toSort[j + 1] = temp

# print sorted list
for x in toSort:
    print(x)

File: Algorithms/C#/BubbleSort.cs
// Bubble Sort with C#
using System;

namespace BubbleSort
{
    class MainClass
    {
        public static void Main()
        {
            // test numbers to sort
            int[] toSort = { 5, 2, 6, 8, 3};

            // perform bubble sort
            for (int i = 0; i < 5; i++)
            {
                for (int j = 0; j < 4; j++)
                {
                    if (toSort[j] > toSort[j + 1])
                    {
                        int temp = toSort[j];
                        toSort[j] = toSort[j + 1];
                        toSort[j + 1] = temp;
                    }
                }
            }

            // print to console results
            for (int i = 0; i < 5; i++)
            {
                Console.WriteLine(toSort[i]);
            }
        }
    }
}
